app/furniture/marketplace/cart.rb

```diff
@@ -37,5 +38,9 @@ def delivery_fee
     def price_total
       product_total + delivery_fee
     end
+
+    def ready_for_delivery?
+      self.delivery_address.present? && self.contact_phone_number.present? ? true: false
```

Comment:
Technically `delivery_address.present? && contact_phone_number.present?` is equivalent, and will return `true` when both are present and `false` when one is not.

Reply:
1. Is this condition coded for the behavior we want? 
2. we want `ready_for_delivery?` to evaluate to `true` when both `delivery_address` and `contact_phone_number` exists right? And if either `delivery_address` or  `contact_phone_number` are missing, it evaluates to `false`, right?

Reply:
nevermind I think I understand now. and Rubocop caught the issue too. my ternary was redundant since `.present` already evaluates to a boolean.

Reply:
fixed in 21395a5729b4858daab4cb3d524a1f6ef0f32e5d